.travis.yml

```diff
@@ -1,8 +1,21 @@
-
 language: python
 python:
   - "3.6.4"
+
+services:
+  - docker
+env:
+  - DOCKER_COMPOSE_VERSION=1.15.0
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
 install:
   - pip install -r requirements.txt
+  - docker-compose -f docker-compose_travis.yml build
```

Comment:
Do we need this line here considering that we are executing `docker-compose up` in the `script` session, and the `up` already builds the image?